### PR TITLE
CURLOPT_ALTSVC_CTRL.3: fix the default

### LINKED ISSUE #4909: CURLOPT_ALTSVC_CTRL man page: DEFAULT is incorrect
### I did this
https://curl.haxx.se/libcurl/c/CURLOPT_ALTSVC_CTRL.html says:
> DEFAULT
> 0. No Alt-Svc treatment. 

But, the code says https://github.com/curl/curl/blob/curl-7_68_0/lib/altsvc.c#L256 :
```
  /* set default behavior */
  asi->flags = CURLALTSVC_H1
#ifdef USE_NGHTTP2
    | CURLALTSVC_H2
#endif
#ifdef ENABLE_QUIC
    | CURLALTSVC_H3
#endif
```
So the documented default is wrong.

### I expected the following
The documentation to be agree with the code :)

### curl/libcurl version

```
$ curl --version
curl 7.68.0 (x86_64-pc-linux-gnu) libcurl/7.68.0 OpenSSL/1.1.1d zlib/1.2.11 brotli/1.0.7 libidn2/2.3.0 libssh2/1.9.0_DEV nghttp2/1.40.0 quiche/0.2.0 librtmp/2.3
Release-Date: 2020-01-08
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtmp rtsp scp sftp smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli HTTP2 HTTP3 HTTPS-proxy IDN IPv6 Largefile libz Metalink NTLM SSL TLS-SRP UnixSockets
```

### operating system

```
# uname -a
Linux irrational 5.4.0-rc6 #2 SMP Fri Nov 8 19:09:49 EST 2019 x86_64 Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz GenuineIntel GNU/Linux
```

### PR DESCRIPTION
**Pull request by bagder**
Reported-by: Craig Andrews
Fixes #4909

### CODE REVIEW

File: docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3 [hunk 1]
```diff
@@ -69,7 +69,9 @@ Accept alternative services offered over HTTP/3. This will only be used if
 libcurl was also built to actually support HTTP/3, otherwise this bit will be
 ignored.
 .SH DEFAULT
-0. No Alt-Svc treatment.
+Depends on what protocol support that is built-in. H1 is always there. H2 if
+libcurl is built with HTTP/2 support and H3 as well if HTTP/3 support is
+present.
```
Comment (jay) [inline docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3]: That is explained above how about just `CURLALTSVC_H1  | CURLALTSVC_H2 | CURLALTSVC_H3`

Comment (jay) [reply docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3]: Ah I see what's happening. CURLOPT_ALTSVC is causes altsvc initialized with default `CURLALTSVC_H1 | CURLALTSVC_H2 | CURLALTSVC_H3`. However altsvc would not be enabled without either CURLOPT_ALTSVC or 
CURLOPT_ALTSVC_CTRL therefore I think 0 is the default. It probably could be explained better though,
"0. No Alt-Svc treatment. If CURLOPT_ALTSVC was set then alt-svc ctrl defaults to CURLALTSVC_H1 | CURLALTSVC_H2 | CURLALTSVC_H3".

Comment (bagder) [reply docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3]: I took a stab at an update. What do you say now?

File: docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3 [hunk 2]
```diff
@@ -69,7 +69,10 @@ Accept alternative services offered over HTTP/3. This will only be used if
 libcurl was also built to actually support HTTP/3, otherwise this bit will be
 ignored.
 .SH DEFAULT
-0. No Alt-Svc treatment.
+Alt-Svc handling is disabled by default. If \fICURLOPT_ALTSVC(3)\fP is set,
+\fCURLOPT_ALTSVC_CTRL(3)\fP has a default value corresponding to CURLALTSVC_H1
```
Comment (jay) [inline docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3]: \fCURLOPT_ALTSVC_CTRL(3)\fP the \f should be \fI. Looks fine otherwise.

Comment (bagder) [reply docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3]: Ack!

**Pull request closed without merging**